src/mixins/nft.js

```diff
@@ -227,19 +228,34 @@ export default {
         this.uiToggleCollectModal();
         this.uiSetCollectedCount(this.userOwnedCount);
         if (balance === '0' || Number(balance) < this.purchaseInfo.totalPrice) {
+          logTrackerEvent(
+            this,
+            'NFT',
+            'NFTCollect-insufficient',
+            this.getAddress,
+            1
+          );
           this.uiSetTxError('INSUFFICIENT_BALANCE');
           this.uiSetTxStatus(TX_STATUS.INSUFFICIENT);
           return;
         }
 
         this.uiSetTxStatus(TX_STATUS.SIGN);
+        logTrackerEvent(this, 'NFT', 'NFTCollect-sendGrant', this.classId, 1);
         const txHash = await sendGrant({
           senderAddress: this.getAddress,
           amountInLIKE: this.purchaseInfo.totalPrice,
           signer: this.getSigner,
         });
```

Comment:
Add logs around the request
```suggestion
        logTrackerEvent(this, 'NFT', 'NFTCollectSendGrantRequested', this.classId, 1);
        const txHash = await sendGrant({
          senderAddress: this.getAddress,
          amountInLIKE: this.purchaseInfo.totalPrice,
          signer: this.getSigner,
        });
       logTrackerEvent(this, 'NFT', 'NFTCollectSendGrantApproved'', this.classId, 1);
```